[2024-09-29T12:45:54.420+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-29T12:45:54.489+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v04.postgres_to_s3 scheduled__2022-10-21T00:00:00+00:00 [queued]>
[2024-09-29T12:45:54.525+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v04.postgres_to_s3 scheduled__2022-10-21T00:00:00+00:00 [queued]>
[2024-09-29T12:45:54.528+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-29T12:45:54.628+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2022-10-21 00:00:00+00:00
[2024-09-29T12:45:54.666+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=23117) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-29T12:45:54.671+0000] {standard_task_runner.py:72} INFO - Started process 23126 to run task
[2024-09-29T12:45:54.671+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v04', 'postgres_to_s3', 'scheduled__2022-10-21T00:00:00+00:00', '--job-id', '1069', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpi5ja18qs']
[2024-09-29T12:45:54.679+0000] {standard_task_runner.py:105} INFO - Job 1069: Subtask postgres_to_s3
[2024-09-29T12:45:54.712+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:45:54.854+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_hooks_v04.postgres_to_s3 scheduled__2022-10-21T00:00:00+00:00 [running]> on host 6851fee35b72
[2024-09-29T12:45:54.858+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-09-29T12:45:55.280+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v04' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2022-10-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-10-21T00:00:00+00:00'
[2024-09-29T12:45:55.299+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-29T12:45:55.441+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/operator_helpers.py:175 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2024-09-29T12:45:55.499+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-29T12:45:55.535+0000] {dag_with_postgres_hooks.py:34} INFO - Saved orders data in text file: dags/get_orders_20221021.txt
[2024-09-29T12:45:55.555+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2024-09-29T12:45:55.561+0000] {connection_wrapper.py:391} INFO - AWS Connection (conn_id='minio_conn', conn_type='aws') credentials retrieved from extra.
[2024-09-29T12:45:55.565+0000] {logging_mixin.py:190} WARNING - <string>:9 AirflowProviderDeprecationWarning: extra['host'] is deprecated and will be removed in a future release. Please set extra['endpoint_url'] instead
[2024-09-29T12:45:56.704+0000] {dag_with_postgres_hooks.py:43} INFO - Orders file /tmp/tmpvyuku56q20221021 has been pushed to S3!
[2024-09-29T12:45:56.719+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-29T12:45:56.776+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-29T12:45:56.782+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v04, task_id=postgres_to_s3, run_id=scheduled__2022-10-21T00:00:00+00:00, execution_date=20221021T000000, start_date=20240929T124554, end_date=20240929T124556
[2024-09-29T12:45:57.065+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-29T12:45:57.240+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-29T12:45:57.246+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
